controller/konnect/reconciler_generic.go

```diff
@@ -603,6 +603,14 @@ func getServiceRef[T constraints.SupportedKonnectEntityType, TEnt constraints.En
 	e TEnt,
 ) mo.Option[configurationv1alpha1.ServiceRef] {
 	switch e := any(e).(type) {
+	case *configurationv1alpha1.KongService,
```

Comment:
I think these `case`s can be removed for simplicity (and reduce potential conflicts).

Reply:
Yeah, my mistake doing rebase. 👍